src/pages/Home/components/FavoritesCitiesWeatherWidget/FavoritesCitiesWeatherWidget.tsx

```diff
@@ -1,11 +1,18 @@
 import { CitiesWeatherWidget } from 'components/CitiesWeatherWidget';
+import { CityWeatherShortcut } from 'components/CityWeatherShortcut';
 import { FavoritesCitiesWeatherWidgetTestIds } from './FavoritesCitiesWeatherWidgetTestIds';
 
 const FavoritesCitiesWeatherWidget = () => {
+  const favouriteCities = ['Warszawa', 'Lublin', 'New York', 'San Francisco', 'Tokyo'];
```

Comment:
In the future please do not use any mocks/dummy data in the implementation, that functionality is a part of other task. There is no need to provide any visual presentation.

Reply:
Should I delete it?